packages/components/Menu/src/MenuItem/MenuItem.tsx

```diff
@@ -54,7 +54,11 @@ export const MenuItem = compose<MenuItemType>({
               {icon && (icon.svgSource || icon.fontSource) && <Slots.fontOrSvgIcon {...icon} />}
             </Slots.iconPlaceholder>
           )}
-          {children && <Slots.content tooltip={tooltipResult}>{children}</Slots.content>}
+          {children && (
```

Comment:
Thanks 😅 Do we need to pass accessible={false} to the icons as well?

Reply:
They didn't show up in the a11y tree so I didn't add them but I guess it wouldn't hurt. I'm not seeing that icons aren't accessible by default.